### DISCUSSION TITLE
Flowify for Argo Workflows is now open source!

### DISCUSSION THREAD

**Discussion by adamchengtkc** [equinor/flowify-workflows-server#35] [Announcements]
Flowify is a no-code portal for creating and running workflows on Argo Workflows. It aims to bring the power of Argo Workflow  beyond developers and data scientists.

- Developers can create sharable components for no-code users
- Users use a drag-and-connect user interface to build workflows from components
- Ability to nest multiple workflows
- Workspaces implementation allows granular access control
- Build on top of Argo Workflow to provide a robust workflow executor

Flowify is not just an Argo Workflows builder:

- Easy to share re-usable components
- Version control for workflows and components
- Designed to make data flow more explicit while Argo Workflows handles execution flow
- No Argo Workflows knowledge is needed for the end user
- Transpilation layer provides flexibility and allows future implementation of other executor

Try it with our local test [environment](https://equinor.github.io/flowify-documentation/run_local/). Contact us if you would like to deploy it on your cluster, we are happy to help. 

Flowify is in early stage. Get in touch if you would like to collaborate or visit our [backlogs](https://github.com/orgs/equinor/projects/269/views/1)

The documentation can be accessed following this link: https://equinor.github.io/flowify-documentation/

**Comment by tooptoop4** [comment]
any screenshots?

**Reply by adamchengtkc** [reply]
Check out the docs https://equinor.github.io/flowify-documentation/

**Reply by adamchengtkc** [reply]
Also shown in this Argo community meeting https://youtu.be/9QBqJ5FgNb0

**Comment by Eva-Music** [comment]
I want to deploy Flowify on my cluster. Can you provide a deployment YAML file or a helm chart?
